includes/class-wc-payments-account.php

```diff
@@ -394,7 +394,15 @@ public static function get_login_url() {
 	 * @return string Stripe account login url.
 	 */
 	public static function get_connect_url() {
-		return wp_nonce_url( add_query_arg( [ 'wcpay-connect' => '1' ] ), 'wcpay-connect' );
+		$connect_url = add_query_arg(
+			[
+				'page' => 'wc-admin',
+				'path' => '/payments/connect',
+			],
+			admin_url( 'admin.php' )
+		);
+
+		return wp_nonce_url( add_query_arg( [ 'wcpay-connect' => '1' ], $connect_url ), 'wcpay-connect' );
```

Comment:
I think `wcpay-connect` may no longer be used.

Reply:
Are you sure? I'm just mimicking the URL that was already in use here.

Reply:
At first, I don't know why we need to make this change. However, it makes sense when I test this PR. Otherwise, the `Connect` button's link here will be something like http://localhost:8082/wp-json/wc/v3/payment_gateways?_locale=user&wcpay-connect=1&_wpnonce=xxxxxxx 


> It actually looks like this is needed to initiate the oauth redirect to Stripe, which is what I intended. This is handled here.
Write

That's right, we should still keep `wcpay-connect` as WCPay uses it to generate the Stripe onboarding link.

However, we do not need to build up $connection_url like this. Instead, using `admin_url( 'admin.php' )` is quite enough and avoids side effects by introducing more parameters.

Note: the below suggestion is for the whole change of this `get_connect_url` method. 

```suggestion
		return wp_nonce_url( add_query_arg( [ 'wcpay-connect' => '1' ], admin_url( 'admin.php' ) ), 'wcpay-connect' );
```